core/chain_indexer.go

```diff
@@ -926,6 +926,44 @@ func (c *ChainIndexer) addOutpointsToIndexer(nodeCtx int, config params.ChainCon
 					outputIndex++
 				}
 			}
+			// This is the conversion reverts from a qi-quai conversion
+		} else if tx.EtxType() == types.ConversionRevertType && tx.To().IsInQuaiLedgerScope() {
```

Comment:
You forgot to do the reorg case, but it's probably ok because I will have to re-do the indexer entirely

Reply:
oh i thought the created utxos are already taken care of in the reorg